docs/advanced-usage/index.md

```diff
@@ -6,7 +6,7 @@ hide:
 
 # :kolena-rocket-20: Advanced Usage
 
-This section contains tutorial documentation for advanced features available in Kolena.
+This section contains tutorial documentation for advanced features for Kolena Workflow.
```

Comment:
Nit: standardize to "Kolena Workflows" (plural)

Reply:
Whoops, this change was supposed to be part of https://github.com/kolenaIO/kolena/pull/431 😅 -- adding to there and renaming to `Kolena Workflows`